core/core-progress-bar.el

```diff
@@ -46,8 +48,11 @@
   (spacemacs//redisplay))
 
 (defun spacemacs/update-progress-bar ()
-  "Update progress bar by incrementing its value by 1.
-Display the progress bar by chunks of size `spacemacs-loading-dots-chunk-threshold'"
+  "When the customization variable
```

Comment:
The first sentence should end in a period.

Just use the original docstring. And add a new line "This has no effect when xxx is nil."

Reply:
I would've reverted to the original docstring if I saw that earlier. The new string should be okay, however.